### PR TITLE
jobs/bodhi-trigger: add reporting to ResultsDB

### PR DESCRIPTION
**Pull request by jlebon**
Call out to the new `resultsdb-report` script in cosa to report test
results back to ResultsDB. Report when first running and then again with
final test results.

Just this alone seems sufficient for the results to show up in Bodhi's
"Automated Tests" tab.

With this, we'll then be able to work on adding policies in Greenwave.

### CODE REVIEW

**Comment by jlebon** [comment]
Requires https://github.com/coreos/coreos-assembler/pull/3656.

File: jobs/bodhi-trigger.Jenkinsfile [hunk 1]
```diff
@@ -203,17 +203,20 @@ if (test_mode) {
...
 
-def test = null
-stage("Test") {
-    test = build(job: 'test-override', propagate: false, wait: true,
-                 parameters: [
-                    string(name: 'STREAM', value: stream),
-                    string(name: 'OVERRIDES', value: msg.update.url),
-                 ])
-}
+cosaPod(cpu: "0.1", kvm: false) {
```
Comment (dustymabe) [inline jobs/bodhi-trigger.Jenkinsfile:206]: 0.1 CPU enough?

**Review by dustymabe — APPROVED** [review]
LGTM

File: jobs/bodhi-trigger.Jenkinsfile (hunk 1 above)
Comment (jlebon) [reply jobs/bodhi-trigger.Jenkinsfile:206]: Yeah, should be plenty. We're just triggering a job and doing two HTTP POSTs essentially. :)

**Comment by jlebon** [comment]
I just removed some extra braces. Merging!

**Pull request merged**